### ISSUE TITLE
Broken message edit

### ISSUE THREAD

**Issue by 3lnc** [fsavje/math-with-slack#38]
STR:
1. Install `math-with-slack`
2. Write message
3. Click ↑ to edit last message
4. Whitespace can't be used while editing

Editing through UI "edit message" works, tho.

Env:
Direct Download 3.2.1 64-bit
Ubuntu 18.04

**Comment by fsavje** [comment]
Thank you for the report. Are you sure this isn't a bug with Slack? The bug pops up also for messages without math in them. Does the bug replicate also without math-with-slack?

**Comment by 3lnc** [comment]
At least on my env — sure, I've checked it before reporting this ;)
Removing `math-with-slack` + restarting slack instantly solves problem.

**Comment by fsavje** [comment]
Thanks -- just checking. This will not be a priority, but I'll look into it when I get a chance.

**Comment by matzhaugenOI** [comment]
Yes, I notice the same thing. The edit can do everything except add whitespace.

**Comment by kmace** [comment]
I have the same issue

**Comment by isnbh0** [comment]
I have the same issue as well :disappointed:

**Comment by rfrowe** [comment]
Please prioritize this more... Every time I want to edit more than a single word in a message I have to write it in an external editor and paste it in

**Comment by fsavje** [comment]
@rfrowe Thank you for letting me know that this is a requested feature. I, however, provide this script free of charge as a courtesy to the community. This is in no way a service I provide to you. If you urgently need this feature, I suggest you spend the weekend implementing it yourself, and then make a pull request.
